docs/architecture/adr/0027-adopt-angular-signals-for-component-state.md

```diff
@@ -0,0 +1,10 @@
+---
+adr: "0027"
+status: "Draft"
+date:
+tags: [client]
+---
+
+# 0026 - Adopt Angular Signals for Component State
+
+Placeholder for documentation
```

Comment:
❓ What's happening with this whole file? Not understanding. You can have one proposed ADR as 27 most likely.